backend/firebase/firebase.tsx

```diff
@@ -1,26 +1,25 @@
 // Import the functions you need from the SDKs you need
 import { FirebaseApp, getApps, initializeApp } from "firebase/app";
 import { getAnalytics } from "firebase/analytics";
-import { getFirestore } from "firebase/firestore";
-import { FacebookAuthProvider, getAuth, GoogleAuthProvider, TwitterAuthProvider } from "firebase/auth";
+import { FacebookAuthProvider, getAuth, GoogleAuthProvider, TwitterAuthProvider, signInWithPopup, signOut } from "firebase/auth";
+import { addDoc, collection, getDocs, getFirestore, query, where } from "firebase/firestore";
 // TODO: Add SDKs for Firebase products that you want to use
 // https://firebase.google.com/docs/web/setup#available-libraries
 
-// Your web app's Firebase configuration
-// For Firebase JS SDK v7.20.0 and later, measurementId is optional
 const firebaseConfig = {
-    apiKey: "AIzaSyAWCKEkLBpYgN_dIJl6ncEHPngL21j3kz4",
-    authDomain: "okb-hope.firebaseapp.com",
-    projectId: "okb-hope",
-    storageBucket: "okb-hope.appspot.com",
-    messagingSenderId: "321787147976",
-    appId: "1:321787147976:web:faae136e03900bd47b97fd",
-    measurementId: "G-V7SV4XK6GC"
-  };
+  apiKey: "AIzaSyAWCKEkLBpYgN_dIJl6ncEHPngL21j3kz4",
+  authDomain: "okb-hope.firebaseapp.com",
+  projectId: "okb-hope",
+  storageBucket: "okb-hope.appspot.com",
+  messagingSenderId: "321787147976",
+  appId: "1:321787147976:web:faae136e03900bd47b97fd",
+  measurementId: "G-V7SV4XK6GC"
+};
```

Comment:
This should definitely not be exposed in your codebase, use [this](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository) as a reference for removing this data from the commit.

Once you do so, store this data in a file called `serviceAccount.json` and then add it to your `.gitignore`, loading the json from that file as your config